apps/web-staking/src/app/components/pool/PoolDropdownComponent.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import { useBlockIp } from "@/app/hooks";
+import { PrimaryButton } from '../../../../../../packages/ui/src/rebrand/buttons/PrimaryButton';
+import { DropdownText, DropdownItem } from "../dropdown/DropdownText";
+import ExternalLinkIcon from "../../../../../../packages/ui/src/rebrand/icons/ExternalLinkIcon";
+import {listOfCountries} from '../../../../../sentry-client-desktop/src/components/blockpass/CountryDropdown';
+
+const PoolDropdownComponent = () => {
+  const [selectedCountry, setSelectedCountry] = useState<string | null>("");
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const { blocked, loading, data } = useBlockIp();
+  
+    function onClickHelper() {
+		 if (selectedCountry) {
+			if (
+				selectedCountry === "China" || 
+				selectedCountry === "Hong Kong" || 
+				selectedCountry === "Republic of North Macedonia" ||
+				selectedCountry === "Turkey" || 
+				selectedCountry === "Ukraine" 	 
+			) {
+				return window.open(`https://verify-with.blockpass.org/?clientId=xai_sentry_node__edd_60145`, "_blank",
+            "noopener noreferrer");
+			} else if (selectedCountry !== "") {
+				return window.open(`https://verify-with.blockpass.org/?clientId=xai_node_007da`, "_blank",
+            "noopener noreferrer");
+			}
+		} else {
+			return
+		}
+	}
+
+  const countries: JSX.Element[] = listOfCountries.filter(item => item.label.toLocaleLowerCase().startsWith(selectedCountry?.toLowerCase()!)).map((item, i, arr) => (
+    <DropdownItem
+      onClick={() => {
+        setSelectedCountry(item.label);
+        setIsOpen(false);
+      }}
+      dropdownOptionsCount={arr.length}
+      key={`sentry-item-${i}`}
+      extraClasses={"hover:!bg-velvetBlack"}
+    >
+      {item.label}
+    </DropdownItem>
+  ));
+
+  const isBlockedCountry = () => { 
+    if((listOfCountries.find(item => item.value === data.country)?.label === selectedCountry) && blocked) {
+      return true
+    }
+    return false
```

Comment:
Is this not what the `blocked` holds from useBlockIp() ?

Reply:
`useBlockIp()` returns boolean only depending on the user's current ip, no matter which country the user chooses, he will still be blocked from continuing the KYC process, so I updated the check, and now the user will not be able to continue the process if his ip is blocked and he chooses a country from the list corresponding to his current location.

Reply:
This seems redundant

Reply:
Done